gc3libs/backends/batch.py

```diff
@@ -124,6 +124,7 @@ def __init__(self, name,
                  # SSH-related options; ignored if `transport` is 'local'
                  ssh_config=None,
                  keyfile=None,
+                 pkey=None,
```

Comment:
There's no need for this if you take `pkey` from `auth.pkey` in line 157.

Reply:
Fixed in 651f88cd9fa2ae19d9f8a1b6cfeab76b718f5b2e